### ISSUE TITLE
[rom,rom_ext] generate UDS and CDI_0 in ROM

### ISSUE THREAD

**Issue by timothytrippel** [lowRISC/opentitan#21552]
For DICE compliance and boot-time optimization, the UDS and CDI_0 attestation keys should be generated in the ROM.
The generation of the CDI_1 attestation keys should remain in the ROM_EXT.

**Comment by andreaskurth** [comment]
@timothytrippel: You added this to the M2 milestone, but shouldn't it be part of the ROM milestone, since no RTL changes are mentioned? If RTL changes are needed, please get in touch and we can assess what needs to be changed before the RTL freeze.

**Comment by timothytrippel** [comment]
Yes, my apologies, meant to add to ROM milestone. Thanks for catching / fixing.

**Comment by moidx** [comment]
This is already completed for the associated milestone `Earlgrey-PROD.ROM`.
